build.gradle.kts

```diff
@@ -9,28 +8,15 @@ buildscript {
     }
 
     dependencies {
-        classpath(appLibs.androidToolsBuildGradle)
-        classpath(appLibs.kotlinGradlePlugin)
-        classpath(appLibs.navigationSafeArgs)
-        classpath(appLibs.googleServices)
-        classpath(appLibs.firebaseGradle)
+        classpath(libs.android.gradlePlugin)
+        classpath(libs.kotlin.gradlePlugin)
+        classpath(libs.google.services)
+        classpath(libs.firebase.gradle)
     }
 }
 
 plugins {
-    id("com.github.ben-manes.versions") version "0.43.0"
-    id("nl.littlerobots.version-catalog-update") version "0.7.0"
-}
-
-allprojects {
-    repositories {
-        google()
-        mavenCentral()
-    }
-}
-
-subprojects {
-    addBasicConfiguration()
+    alias(libs.plugins.versions)
```

Comment:
Not using the `version-catalog-update` anymore as it wouldn't behave as expected (not updating the dependencies, removing needed dependencies) but also because we usually don't want to update libraries automatically. Right now, we can still generate a report using the `dependencyUpdates` task and update the wanted dependencies manually.